src/drf_json_api_utils/sql_alchemy/factory.py

```diff
@@ -152,6 +152,11 @@ def __warn_if_method_not_available(self, method: str):
                 f'You\'ve set a lifecycle callback for resource {self._resource_name}, '
                 f'which doesn\'t allow it\'s respective HTTP method through `allowed_methods`.')
 
+    def _capture_exception(self, e: Any):
+        traceback.print_exc()
```

Comment:
Lets remove this, instead if the user of the library would like to print the traceback they can do so inside the callback